solutions/security/get-started/get-started-detect-with-siem.md

```diff
@@ -0,0 +1,179 @@
+5. Select a configuration preset. Each preset comes with different default settings for {{agent}}, which you can further customize later by [configuring the {{elastic-defend}} integration policy](/solutions/security/configure-elastic-defend/configure-an-integration-policy-for-elastic-defend.md). For optimal endpoint protection, we recommend selecting **Complete EDR (Endpoint, Detection & Response)**. 
+6. Enter a name for the agent policy in the **New agent policy name** field.
+7. Click **Save and continue**. Next, click **Add {{agent}} to your hosts**. 
+:::{image} /solutions/images/security-gs-siem-defend-flyout.png
+:alt: Elastic Defend configuration
+:screenshot:
+:::
+::::
+
+::::{step} Add the Elastic Agent
+
+[{{agent}}](/reference/fleet/index.md#elastic-agent) is a single, unified way to add monitoring for logs, metrics, and other types of data to a host. You'll need to install this component so it can monitor any malicious activity on your hosts. 
+
+:::{dropdown} Steps to add {{agent}}
+1. In the **Add agent** flyout that appears after you install the {{elastic-defend}} integration, you'll see the policy selected that you previously added. Leave the default enrollment token selected. 
+2. Ensure that the **Enroll in {{fleet}}** option is selected. {{elastic-defend}} cannot be integrated with {{agent}} in standalone mode.
+3. Select the appropriate platform or operating system for the host on which you're installing the agent, then copy the provided commands.
+4.  On the host, open a command-line interface and navigate to the directory where you want to install {{agent}}. Paste and run the commands from {{fleet}} to download, extract, enroll, and start {{agent}}.
+5. (Optional) Return to the **Add agent** flyout, and observe the **Confirm agent enrollment** and **Confirm incoming data** steps automatically checking the host connection. It may take a few minutes for data to arrive in {{es}}.
+6. (Optional) After you have enrolled the {{agent}} on your host, you can click **View enrolled agents** to access the list of agents enrolled in {{fleet}}. Otherwise, select **Close**.
+
+    The host will now appear on the **Endpoints** page in the {{security-app}} (**Assets** → **Endpoints**). It may take another minute or two for endpoint data to appear in {{elastic-sec}}.
+
+:::{important}
+If you’re using macOS, some versions may require you to grant {{elastic-endpoint}} Full Disk Access to different kernels, system extensions, or files. Refer to [Elastic Defend requirements](/solutions/security/configure-elastic-defend/elastic-defend-requirements.md) for more information.
+:::
+:::
+::::
+
+::::{step} Modify policy configuration settings
+
+After you install the {{agent}} with {{elastic-defend}}, the Endpoint Security ({{elastic-defend}}) detection rule is automatically enabled and can generate either detection or protection alerts
```

Comment:
Dot missing at the end of this sentence.

Reply:
Good catch! Fixed in 21bbbe16b20524e694fd9715ab115a563de7433f